site/app/templates/polls/ViewPoll.twig

```diff
@@ -22,34 +22,35 @@
                 <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
                 <input type="hidden" name="poll_id" value="{{ poll.getId() }}"/>
                 <fieldset>
+                    <table>
```

Comment:
Should the elements inside `<table>` and `</table>` be indented?

Reply:
They automatically get indented when placed as td in a tr as now the text has its own cell.